src/app/ubs-user/ubs-user-orders-list/ubs-user-orders-list.component.ts

```diff
@@ -33,6 +33,8 @@ export class UbsUserOrdersListComponent {
     this.orders.forEach((order) => {
       if (order.generalOrderInfo.id === id) {
```

Comment:
Can it be so?
order.extend = order.generalOrderInfo.id === id ? !order.extend : false;

Reply:
Good remark. I fixed it.